dali/pipeline/data/views.h

```diff
@@ -103,6 +103,8 @@ view(TensorList<Backend> &data) {
 template <typename T, int ndim = kernels::DynamicDimensions, typename Backend>
 kernels::TensorListView<detail::storage_tag_map_t<Backend>, T, ndim>
 view(const TensorList<Backend> &data) {
+  static_assert(std::is_const<T>::value,
+                "Cannot create a non-const view of a `const TensorList<>`. Missing `const` in T?");  // NOLINT
```

Comment:
What about 
```suggestion
                "Cannot create a non-const view of a `const TensorList<>`. "
                "Missing `const` in T?");
```
instead of using `// NOLINT` here.

Reply:
Done